src/App.js

```diff
@@ -1,16 +1,60 @@
-import React from 'react';
+import React, { Component } from 'react'
+//import './App.css'
+import TodoList from './components/TodoComponents/TodoList'
+import TodoForm from './components/TodoComponents/TodoForm'
 
-class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+class App extends Component {
+  inputElement = React.createRef()
+  constructor() {
+    super()
+    this.state = {
+      items: [],
+      currentItem: {
+        text: '',
+        key: '',
+      },
+    }
+  }
+  deleteItem = key => {
+    const filteredItems = this.state.items.filter(item => {
+      return item.key !== key
```

Comment:
This is a good way of handling deleting items from state. However, the MVP was for you to be able to set the state of each todo so that a "completed" property is set to true or false when each item is clicked. Then you are supposed to be able to clear all of the "completed" todo items. Currently, this functionality isn't present in your app.

Please take a look at this repo: https://codesandbox.io/s/github/lilmatchamonster/React-Todo

and try to refactor your todo list so that it handles completing and deleting todos in a similar way. The example above is one of the only projects that I've seen that got the functionality completely right.